models2/Wiki.es6.js

```diff
@@ -0,0 +1,32 @@
+import RedditModel from './RedditModel';
+import Record from '../apiBase/Record';
+
+import { WIKI } from './thingTypes';
+
+const T = RedditModel.Types;
+
+export default class Wiki extends RedditModel {
+  static type = WIKI;
+
+  static PROPERTIES = {
+    contentHTML: T.string,
+    contentMD: T.string,
+    path: T.string,
+    mayRevise: T.bool,
+    revisionBy: T.nop,
+    revisionDate: T.number,
+  }
+
+  static API_ALIASES = {
+    content_html: 'contentHTML',
+    content_md: 'contentMD',
+    may_revise: 'mayRevise',
+    path: 'path',
```

Comment:
I would remove path here -- though this works I would keep aliases only as properties that renaming (read underscores to camelCase) here